docs/faq.md

```diff
@@ -15,6 +19,31 @@ using standard HTTP compression negotiation and eliminating binary framing from
 the body, the Connect protocol lets us make unary RPCs with web browsers, cURL,
 or any other HTTP client.
 
+### How do I serve gRPC services when they are already written in a language that doesn't have a Connect implementation?
+
+Envoy is a popular and battle-tested proxy that supports gRPC very well. It's
+widely used, sometimes under the hood, of projects like Contour or Istio.
+
+Envoy v1.26 ships with a Connect-gRPC bridge that allows clients to speak the
+Connect protocol (including GET requests) to existing gRPC servers. You can find
+a demo here: https://github.com/connectrpc/envoy-demo
+
+### How do I reliably call a server streaming RPC from a web browser?
+
+The answer is highly dependent on all of the networking parties involved. Generally,
+make sure that your server or your infrastructure does not apply timeouts within
+the expected duration of calls. If possible, pre-empt timeouts by setting short
+deadlines and by repeating the call when the deadline is exceeded. Read the
+[streaming docs](go/streaming.md) for the Go implementation to get an idea of
+the implications.
+
+### How do I proxy the Connect protocol through NGINX?
+
+NGINX doesn't support end-to-end HTTP/2, so it's not possible to proxy the
+Connect protocol through NGINX properly. Some features such as bidirectional
+streaming will never work. Suitable reverse proxies include Envoy Proxy and
```

Comment:
I'd like this answer to more clearly state that _unary_ RPCs will work fine, but streaming RPCs will not.